docs/source/guide/explanation/algorithms/object_detection/object_detection.rst

```diff
@@ -133,7 +137,7 @@ Overall, OpenVINO™ Training Extensions utilizes powerful techniques for improv
 
 Please, refer to the :doc:`tutorial <../../../tutorials/advanced/semi_sl>` how to train semi supervised learning.
 
-In the table below the mAP on toy data sample from `COCO <https://cocodataset.org/#home>`_ dataset using our pipeline is presented.
+In the table below the mAP on toy data sample from `COCO <https://cocodataset.org/#home>`__ dataset using our pipeline is presented.
```

Comment:
is this double underscore expected?

Reply:
Yes, in other way I get a warning that I make 2 links to one COCO source (it's cited in Self-SL as well)